lib/tasks/db.rake

```diff
@@ -0,0 +1,6 @@
+namespace :db do
```

Comment:
I would rename my namespace something different since `rake db` already exists within rails. I'm not sure that is your issue, but that is the first thing I would change.

Next, you don't necessarily need to drop everything. Mainly you just want to clear out the data that is in them so I might just use 'destory_all` or 'delete_all` on each of the models before running the import task. 
So it would look something like `Customer.delete_all` etc.

Reply:
Thank you for the direction! That did it!